### ISSUE TITLE
[BUG] [pre-commit] Type tag 'nunjucks' is not recognized.  Try upgrading identify and pre-commit?

### ISSUE THREAD

**Issue by frague59** [Riverside-Healthcare/djLint#366]
Hi, 

I've an issue with pre-commit hook on a django project.

- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Debian Bookworm/Sid
 - Python Version : 3.9.9
 - djLint Version pipx: 1.12.1
 - template language: django

## Issue
I'm adding a pre-commit hook, following the docs:

```yaml
  - repo: https://github.com/Riverside-Healthcare/djLint
    rev: 'v1.12.1'
    hooks:
      - id: djlint-django
```

I've an error when starting the `pre-commit` command:

```bash
pre-commit run --all

An error has occurred: InvalidManifestError: 
==> File /home/<username>/.cache/pre-commit/repoyca8lunl/.pre-commit-hooks.yaml
==> At Hook(id='djlint-nunjucks')
==> At key: types_or
==> At index 0
=====> Type tag 'nunjucks' is not recognized.  Try upgrading identify and pre-commit?
Check the log at /home/<username>/.cache/pre-commit/pre-commit.log
```

Thanks !

**Comment by christopherpickering** [comment]
@frague59 thanks! nunjucks is a valid type in the `identify` library, did you already try upgrading that and pre-commit?

I'm on pre-commit 2.19 and can't get it to reproduce. What version are you on?

It might be worth mentioning over at pre-commit. There is a similar issue there, but I think a different problem. https://github.com/pre-commit/pre-commit/issues/2373

**Comment by frague59** [comment]
@christopherpickering Thanks for your help ! 

I've upgraded both `pre-commit` and `identify`, it's OK !

```bash
$ pipx upgrade pre-commit
$ pipx runpip pre-commit install --upgrade identify
$ pre-commit --version 
pre-commit 2.20.0
```

Maybe you could add comment on install about that...

Thanks for your help !

**Comment by christopherpickering** [comment]
nice, thanks!
